utils/bazel/llvm-project-overlay/libc/libc_build_rules.bzl

```diff
@@ -108,16 +108,23 @@ def libc_function(
         name = libc_internal_target(name),
         srcs = srcs,
         copts = copts,
+        local_defines = local_defines,
         **kwargs
     )
 
     # This second target is the llvm libc C function with either a default or hidden visibility.
     # All other functions are hidden.
-    func_attrs = ["__attribute__((visibility(\"default\")))"]
-    if weak:
-        func_attrs = func_attrs + ["__attribute__((weak))"]
-    local_defines = local_defines or ["LIBC_COPT_PUBLIC_PACKAGING"]
-    local_defines = local_defines + ["LLVM_LIBC_FUNCTION_ATTR='%s'" % " ".join(func_attrs)]
+    global_func_attrs = [
```

Comment:
Remove temp variable for clarity - you can just use `LLVM_LIBC_FUNCTION_ATTR='[[gnu:visibility(\"default\")]]'` below instead.

Reply:
Done.